### ISSUE TITLE
Garbled text in --help message

### ISSUE THREAD

**Issue by agraef** [davidmoreno/rtpmidid#3]
Looks like there's a line end (\n) missing here:
https://github.com/davidmoreno/rtpmidid/blob/c77c69c00987ba7874139979ca72db6cec256be0/src/config.cpp#L40

Also, maybe reword to something like "This option is the default, no need to specify it explicitly." (and perhaps make that a new line, so that the option isn't broken awkwardly in an 80 characters terminal).

**Comment by agraef** [comment]
Just for the record, here's how the output of `--help` looks like in my terminal:

~~~
Options:
  --version           Show version
  --help              Show this help
  --name <name>       Forces a rtpmidi name
  --port <port>       Opens local port as server. Default 5400. Can set several.
  --connect <address> Connects the given address. This is default, no need for -
-connect  hostname            Connects to hostname:5400 port using rtpmidi
  hostname:port       Connects to a hostname on a given port
  name:hostname:port  Connects to a hostname on a given port and forces a name f
or alsaseq
~~~

The last line could maybe also be broken into to become more readable.

Just a suggestion, of course this is only a cosmetic issue.

**Comment by TheTechnobear** [comment]
this can be closed - due to commit fbabaecf76b3d9445263be03e687bdd0f4d908d7
